src/cassandra/column.rs

```diff
@@ -124,22 +119,37 @@ impl Debug for Column {
             CASS_VALUE_TYPE_TIMEUUID => write!(f, "TIMEUUID Cassandra type"),
             CASS_VALUE_TYPE_INET => write!(f, "INET Cassandra type"),
             CASS_VALUE_TYPE_LIST => {
-                for item in self.set_iter().expect("a list should always be able to return a set iterator") {
-                    write!(f, "LIST {:?}", item)?
+                match self.set_iter() {
```

Comment:
Don't worry too much about these - I make a better fix to them in the next PR, https://github.com/Metaswitch/cassandra-rs/pull/23